ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/core/StandardComponentInitializer.java

```diff
@@ -497,7 +498,11 @@ public void initialize() {
         loaderToolbarGroup.add(loaderAction);
 
         // Define hot-keys
-        keyBinding.getGlobal().addKey(new KeyBuilder().action().alt().charCode('n').build(), "navigateToFile");
+        if(UserAgent.isMac()){
```

Comment:
I don't think so,  for me it's more difficult to understand that for mac uses differed hotkey